assets/js/constants/i18n/repositoriesMessages.js

```diff
@@ -0,0 +1,149 @@
+const repositoriesPageMessages = {
+    'languages': {
+        'en': {
+            'repositories-title1': 'UNCode | Repositorios',
+            'repositories-header1': 'Repositories',
+            'repositories-description1': 'You can see the used repositories below.',
+            'repositories-github': 'Go GitHub',
+            'repository1-description1': 'This is the main UNCode repository',
+            'repository1-description2': 'UNCode is an online platform - developed on ' +
+                                        '<a href="https://github.com/UCL-INGI/INGInious" target="_blank">INGIniois (v0.5)</a>' +
+                                        '- to practice programming exercises and perform automatic grading ' +
+                                        'of code made by the students to help them in their learning processes.',
+            'repository1-description3': 'UNCode is actually used by the Universidad Nacional de Colombia, ' +
+                                        'Bogotá headquarters, to evaluate the written code by the students in multiple programming ' +
+                                        'languages as C / C ++, Java and Python 3; also, the UNCode judge can be used in hardware ' +
+                                        'description languages such as VHDL and Verilog, or with Jupyter Notebooks tasks. ' +
+                                        'For this reason, UNCode has been used in different university courses as: ' +
+                                        'basic computer programming, data structures, machine learning, programming languages, ' +
+                                        'quantum programming, etc.',
+            'repository1-description4': 'UNCode have a backend that handles the interaction with Docker and ' +
+                                        'the grader, and a frontend where the students can upload their codes in a simple and ' +
+                                        'pleasant interface. Also, the frontend includes an administrator interface to allows ' +
+                                        'teachers and tutors check their students progress and modify the tasks on a simple way.',
+            'repository1-description5': 'Both backend and frontend are independent of each other. The backend, ' +
+                                        'in particular, has been created to be used as if it was a library.',
+            'repository1-description6': 'Another interesting feature of UNCode is that it can be used as an external grader for ' +
+                                        'Learning Management Systems (LMS), as Moodle or OpenEDX.',
```

Comment:
```suggestion
            'repository1-description6': 'Another interesting feature of UNCode is that it can be used as an external grader for ' +
                                        'Learning Management Systems (LMS), like Moodle or OpenEDX.',
```